### ISSUE TITLE
base64enc not available

### ISSUE THREAD

**Issue by methodds** [RevolutionAnalytics/RRO#277]
Hi,

at the moment I'm trying to setup RRO with the  [IR-Kernel](https://github.com/IRkernel/IRkernel) for jupyter notebook. During the installation procedure (see [here](http://irkernel.github.io/installation/)) I encounter a problem:

```
ERROR: dependency ‘base64enc’ is not available for package ‘IRdisplay’
```
Trying to install the package manually throws the error:
 ``package ‘base64enc’ is not available (for R version 3.2.3) ``.

Do you have any idea what is going on here? I'm using Linux Mint and the latest RRO version.

**Comment by sfweller** [comment]
I am not sure what is going on, the 'base6enc' package is available for MRO 3.2.3 and is downloadable from the mran website.

My suggestion would be that you manually download the package source tarball from the mran website,
which is:  

https://mran.revolutionanallytics.com/snapshot/2016-01-01/src/contrib/base64enc_0.1-3.tar.gz

After downloading the tarball you can install it with:

R CMD INSTALL base64enc_0.1-3.tar.gz


Stephen Weller
Microsoft R Open Technical Support

**Comment by j-martens** [comment]
There is a small typo above. Try: https://mran.revolutionanalytics.com/snapshot/2016-01-01/src/contrib/base64enc_0.1-3.tar.gz
